s3fs/core.py

```diff
@@ -652,6 +681,18 @@ def close(self):
                 self.s3.s3.put_object(Bucket=self.bucket, Key=self.key)
             self.s3._ls(self.bucket, refresh=True)
 
+    def readable(self):
```

Comment:
They have to be callable to be compatible with the `io` base classes (which surprised me that they weren't properties).